### DISCUSSION TITLE
Is WALC dead....nothing in 6 months

### DISCUSSION THREAD

**Discussion by bkmo** [cstayyab/WALC#131] [General]
This project has stalled for over 6 months now. If not for the work of [Zzombiee2361](https://github.com/Zzombiee2361) we wouldn't have a working copy.

**Comment by daffamumtaz2361** [comment]
Glad you asked. I have been busy with WALC on [my fork](https://github.com/Zzombiee2361/WALC) rewriting the code, moving away from menu to a vue based dashboard page, some initial work to support multi instance, bug fixes, etc. There's still a lot I would do but I settled on recreating the offline page and updating the README, then release it. This is quite a big change and unfortunately the repo owner went silence so I can't ask him to collaborate. And if I do make a release, I won't be able to update the snap version since I have no access to the snap store account.

**Reply by cstayyab** [reply]
Hey!! I saw that you have done some work in Vue.js on your fork since we last spoke. And I would understand if you want to keep it on your fork and release it from there as I haven't been able to contribute much to the project since last release. If you want I can transfer the ownership of snap in snapstore and also redirect users from this repo to your fork using a deprecation notice in README file.

**Reply by bkmo** [reply]
Good to see you are still checking in cstayyab.  Zzombiee2361 has been doing a great job keeping us with a working copy. It is great to see your offer to pass the torch to him.
